### PR TITLE
fix: use project name when adding project via url

### LINKED ISSUE #759: Project name is lost when adding a project via project configurator
**Description**
On https://www.aem.live/tools/sidekick/, only the github url is used to add the project. The project name is ignored. This also happens when sharing a project configuration.

**To Reproduce**
1. Go to https://www.aem.live/tools/sidekick/
2. Specify the github url and project name X
3. Click "go" then "add project"
4. Go to sidekick options in the browser. The project name X is not persisted for the project.

**Expected behavior**
Specified project name on https://www.aem.live/tools/sidekick/ should be used as part of the project configuration

### PR DESCRIPTION
**Pull request by icaraps**
Fixes #759

### CODE REVIEW

**Review by rofe — APPROVED** [review]

File: src/extension/background.js [hunk 1]
```diff
@@ -66,10 +66,16 @@ async function getConfigFromTabUrl(tabUrl) {
   const share = getShareSettings(tabUrl);
   if (share.giturl) {
     // share url
-    return getGitHubSettings(share.giturl);
+    return {
+      ...share,
```
Comment (rofe) [inline src/extension/background.js:70]: @icaraps I think the `giturl` should be removed from `share` here.

**Review by rofe — CHANGES_REQUESTED** [review]

File: src/extension/background.js (hunk 1 above)
Comment (icaraps) [reply src/extension/background.js:70]: Oh why ? Is the config malformed ? When making these changes, I wasn't sure how the config object should look like:
> @returns {object} The config object

Comment (rofe) [reply src/extension/background.js:70]: Just a cleanliness issue. It is no longer used downstream.

Comment (icaraps) [reply src/extension/background.js:70]: Ok I undefined its value.

**Pull request merged**